features/topology-map/org.opennms.features.topology.app/src/main/java/org/opennms/features/topology/app/internal/operations/RedoLayoutOperation.java

```diff
@@ -41,8 +41,8 @@
 	@Override
     public void execute(List<VertexRef> targets, OperationContext operationContext) {
         if (operationContext != null && operationContext.getGraphContainer() != null) {
-            //operationContext.getGraphContainer().getBaseTopology().refresh();
-            //operationContext.getGraphContainer().setDirty(true);
+            operationContext.getGraphContainer().getBaseTopology().refresh();
```

Comment:
Can we then also change the TopologyUI auto refresher logic to use the "RedoLayoutOperation" instead?
See https://github.com/OpenNMS/opennms/blob/develop/features/topology-map/org.opennms.features.topology.app/src/main/java/org/opennms/features/topology/app/internal/TopologyUI.java#L166